config/repo-config.yml

```diff
@@ -0,0 +1,102 @@
+services:
+  notary_url: "http://notary-service.build.10gen.cc:5000"
+
+templates:
+  deb:
+    org: |
+      Origin: mongodb
+      Label: mongodb
+      Suite: {{ .CodeName }}
+      Codename: {{ .CodeName }}/mongodb-org
+      Architectures: {{ .Architectures }}
+      Components: {{ .Component }}
+      Description: MongoDB packages
+    enterprise: |
+      Origin: mongodb
+      Label: mongodb
+      Suite: {{ .CodeName }}
+      Codename: {{ .CodeName }}/mongodb-enterprise
+      Architectures: {{ .Architectures }}
+      Components: {{ .Component }}
+      Description: MongoDB packages
+  index_page: |
+    <!DOCTYPE html>
+    <html>
+     <head>
+       <title>{{ .Title }}</title>
+     </head>
+     <body>
+       <table>
+        <tr><td>
+          <h1>{{ .Title }}</h1>
+        </td></tr>
+        <tr><td>
+          <hr>
+        </td></tr>
+        <tr><td>
+          <a href='..'>Parent Directory</a>
+        </td></tr>
+        {{ range $fn := .Files }}
+        <tr><td>
+          <a href='{{ $fn }}'>{{ $fn }}</a>
+        </td></tr>
+        {{ end }}
+        <tr><td>
+          <hr>
+        </td></tr>
+        <tr><td>
+          <address>{{ .RepoName }}</address>
+        </td></tr>
+      </table>
+     </body>
+    </html>
+repos:
+
+####################
+#
+# Community Repos:
+#
+####################
+  - name: debian92
+    type: deb
+    code_name: "stretch"
+    bucket: repo.mongodb.org
+    edition: org
+    component: main
+    architectures:
+      - amd64
+    repos:
+      - apt/debian/dists/stretch/mongodb-org
+
+  - name: debian10
+    type: deb
+    code_name: "buster"
+    bucket: repo.mongodb.org
+    edition: org
+    component: main
+    architectures:
+      - amd64
```

Comment:
With the `repo-config`'s `architectures` field, I am not sure whether I have to send all possible architectures with the `curator` CLI. Suppose the `architectures` here are:
```yml
    architectures:
      - amd64
      - ppc64el
      - s390x
      - i386
```
Do I have to send all of them? Or none at all is also ok?

Additionally, I haven't seen any of the distros to have `x86_64`, is that an implied architecture that all of them will build to?